source/features/issues-navigation.tsx

```diff
@@ -0,0 +1,46 @@
+import features from '../libs/features';
+
+/**
+ * TODO: find a way o parse this correctly, maybe ask a GitHub developer how this is parsed?
+ * ex: `is:issue is:open sort:updated-desc label:"change request" show branch`
```

Comment:
No need to parse it, you can pass it directly to the Search API (GraphQL)

Reply:
Hmmm, didn't found a way to pass query to GraphQL API.
In v3 I see that query can be passed as is.
https://developer.github.com/v3/search/#search-issues-and-pull-requests

Reply:
Ok, I need to learn better GraphQL 🙂 a search query exists

<details>
<summary>query example</summary>

```graphql
{
  search(last: 3, type: ISSUE, query: "user:sindresorhus repo:refined-github") {
    issueCount
    pageInfo {
      hasNextPage
      endCursor
    }
    edges {
      node {
        ... on Issue {
          createdAt
          title
          url
        }
      }
    }
  }
}
```

</details>